credentialz/credentialz.proto

```diff
@@ -510,11 +510,24 @@ message UserPolicy {
 message AuthorizedUsersResponse {
 }
 
+// Hash type's
+enum HashType {
+  HASH_TYPE_UNSPECIFIED = 0;
+  HASH_TYPE_CRYPT_MD5 = 1;
+  HASH_TYPE_CRYPT_SHA_2_512 = 2;
+}
```

Comment:
Is this enum going to be used in other messages than the CryptoHash message? If not then please move it into it.

Reply:
Addressed this